pkgs/tools/misc/feed2mail/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, python2Packages, fetchFromGitHub,
+configurationString ? null
+}:
+
+
+assert lib.asserts.assertMsg ( configurationString != null )
+  "You need to pass a configuration string to the feed2mail  package";
```

Comment:
"You need to pass a configuration string to the feed2mail  package"; -> "You need to pass a configuration string to the feed2mail package";

Reply:
BTW, I find really strange that you need to pass the `configurationString` for this package to build. I think maybe setting a default config string would be better?

Reply:
We can use the contents of this file as an example:
https://github.com/jonashaag/feed2mail/blob/master/example_config.py

Reply:
Packages shouldn't require any config to build.

Reply:
Yeah, this is why I am suggesting setting a default config, so at least the package could build without any change from user.

However, there are other packages have config files that are meant to be passed during build, i.e.: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/window-managers/dwm/default.nix#L24-L25